apps/meteor/server/models/raw/Rooms.js

```diff
@@ -309,6 +329,7 @@ export class RoomsRaw extends BaseRaw {
 				$in: groupsToAccept,
 			},
 			name: nameRegex,
+			federated: { $exists: false },
```

Comment:
In my local instance all my private groups had `federated: false`; This change made me unable to add any of them to a team.
I don't know why that flag exist on them, but I assume it was used by some other feature at some point.

Reply:
It's indeed a bug, the front end was always sending the federated property, even if it was false. But besides that, I should also add the validation to cover that case. It was fixed in the next commit. Thanks for noticing that.